Makefile

```diff
@@ -18,10 +18,8 @@ ifeq ($(RP_IMAGE_ACR),arointsvc)
 	REGISTRY = arointsvc.azurecr.io
 else ifeq ($(RP_IMAGE_ACR),arosvc)
 	REGISTRY = arosvc.azurecr.io
-else ifeq ($(RP_IMAGE_ACR),)
-	REGISTRY = registry.access.redhat.com
 else
-	REGISTRY = $(RP_IMAGE_ACR)
+	REGISTRY = registry.access.redhat.com
```

Comment:
Why this change?  This drifts from our ARO-RP Makefile.  

Essentially if you have an override set, it will use that, otherwise it will default to registry.access.redhat.com

Reply:
it contradicts with the line 4, if $(RP_IMAGE_ACR) is empty, then $ARO_IMAGE_BASE would be .azurecr.io/aroinstaller. At the same time we gave a condition where we are expecting $(RP_IMAGE_ACR) to be empty in the else if condition which I removed.
Also to build the aro-installer locally `ubi-minimal` image is used as the base image which is publicly available in the registry.access.redhat.com registry.

Reply:
makes sense to me.  want to change this in ARO-RP as well then?

Reply:
@bennerv 
Raised one here: https://github.com/Azure/ARO-RP/pull/3299